app/Resources/views/base.html.twig

```diff
@@ -7,16 +7,28 @@
         {% block stylesheets %}
             <link href="{{ asset('bundles/oag/stylesheets/normalize.css') }}" rel="stylesheet" />
             <link href="{{ asset('bundles/oag/stylesheets/main.css') }}" rel="stylesheet" />
+            <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
```

Comment:
Do we still need this external dependency if the font files have been added locally?

Reply:
Nope